VK_App/VK_App/NetworkService/NetworkService.swift

```diff
@@ -35,9 +35,34 @@ final class NetworkService {
         case photo = "/photos.getAll"
         case newsFeed = "/newsfeed.get"
     }
-    
+
     // MARK: - Public Method
 
+    func getGroupsRequest(request: Methods, ownerIdValue: Int = Constants.ownerIDValue) -> DataRequest {
+        let url = "\(Constants.baseUrl)\(Constants.path)\(request.rawValue)"
+        let parameters: Parameters = [
+            Constants.accessToken: Session.shared.token,
+            Constants.extended: Constants.extendedValue,
+            Constants.vKey: Constants.vValue
+        ]
+        return AF.request(url, parameters: parameters)
+    }
+
+    func getGroups() {
+        let queue = OperationQueue()
+        let request = getGroupsRequest(request: .groups)
+        let getDataOperation = GetDataOperation(request: request)
+        queue.addOperation(getDataOperation)
+
+        let parseData = ParseDataOperation()
+        parseData.addDependency(getDataOperation)
+        queue.addOperation(parseData)
+
+        let saveToRealm = SaveToRealm()
+        saveToRealm.addDependency(parseData)
+        OperationQueue.main.addOperation(saveToRealm)
```

Comment:
```suggestion
        OperationQueue.main.addOperation(saveToRealm)
```
saveToRealm в главном потоке из-за того, что в других потоках не работает?